packages/containers/src/DeleteResource/deleteResource.sagas.js

```diff
@@ -23,6 +22,20 @@ export function buildHttpDelete(uri, labelResource, responseType) {
 	});
 }
 
+/**
+ * for a resourceType and an id find and return the resource
+ * @param {object} state
+ * @param {String} resourceType
+ * @param {String} id
+ */
+export function findResource(state, resourceType, id) {
```

Comment:
Don't you think that it will be better to have this kind of selectors somewhere in cmf / collections instead ? Seems this chunk of code can be reusable.